src/outputs/parthenon_hdf5.cpp

```diff
@@ -90,8 +90,8 @@ void PHDF5Output::WriteOutputFileImpl(Mesh *pm, ParameterInput *pin, SimTime *tm
   auto const nx2 = out_jb.e - out_jb.s + 1;
   auto const nx3 = out_kb.e - out_kb.s + 1;
 
-  const int rootLevel = pm->GetRootLevel();
-  const int max_level = pm->GetCurrentLevel() - rootLevel;
+  const int rootLevel = pm->GetAthenaCompositeRootLevel();
```

Comment:
For lack of a better idea, I called all of the old notions about the mesh `Athena*`.  I guess we can probably always define a root level though, so maybe it is better to just call this `GetRootLevel()` as before.

Reply:
Ah, I actually wasn't thinking about this correctly before. I think this root level is unique to the Athena style grid.

Reply:
What about the `Athena` style grid is unique here?

Reply:
Per our conversation, I am going to leave this as `Athena` since it is consistent with the levels of the `Athena` logical locations that are used for output.